Source/GLPK.swift

```diff
@@ -329,7 +329,11 @@ public class SwiftGLPK {
     var functionValue: Double {
         return glp_get_obj_val(problemPointer)
     }
-    
+
+    /// Returns the solution variable for given index
+    public func getSolutionVariable(i: Int32) -> Double {
```

Comment:
In line with the latest Swift guidelines, I think the following would be more appropriate:

```swift
public func getSolutionVariable(atIndex i: Int32) -> Double {
    // ...
}
```